sdk/ml/azure-ai-ml/tests/batch_services/e2etests/test_batch_deployment.py

```diff
@@ -129,6 +129,7 @@ def test_batch_deployment_dependency_label_resolution(self, client: MLClient, ra
         )
         assert resolved_model.asset_name == model_name and resolved_model.asset_version == model_versions[-1]
 
+    @pytest.mark.skip(reason="Test timing out consistently - can't re-record, so disabling it")
```

Comment:
If you have not already, consider filing a bug to fix it for tracking!

Reply:
I will before I merge!

Reply:
Just committed an update adding a TODO and work item #